service/db.go

```diff
@@ -232,3 +234,40 @@ func deleteUser(uuid string) error {
 
 	return nil
 }
+
+func getUserRow(uuid string) (*pb.User, error) {
```

Comment:
This service is very identical to `DeleteUser` Service.
The following lines of code is similar except for different error message:

```	
        if req == nil {
		return nil, status.Error(codes.InvalidArgument, errNilRequestUser.Error())
	}

	if err := refreshDBConnection(); err != nil {
		return nil, status.Error(codes.Unknown, err.Error())
	}

	// get User Object
	user := req.GetUser()
	if user == nil {
		logger.Error(errNilRequestUser.Error())
		return nil, status.Error(codes.InvalidArgument, errNilRequestUser.Error())
	}

	// validate user id
	uuid := user.GetUuid()
	if err := validateUUID(uuid); err != nil {
		logger.Error("GetUser validating uuid: ", err.Error())
		return nil, status.Error(codes.InvalidArgument, err.Error())
	}

	// check uuid exists
	exists, err := checkUserExists(uuid)
	if err != nil {
		logger.Error("GetUser checkUserExists:", err.Error())
		return nil, status.Error(codes.Internal, err.Error())
	}

	if !exists {
		logger.Error(errDoesNotExistUUID.Error())
		return nil, status.Error(codes.NotFound, errDoesNotExistUUID.Error())
	}
```

I am also thinking for UpdateUser, I would also need these exact lines of code before updating. Should I put this into its own utility function?

Reply:
I think we can get away of not checking if the user exists for updating user